components/home/menu/calendar/Screens/LoginScreen/index.js

```diff
@@ -0,0 +1,96 @@
+import React, { Component } from 'react';
+import { View, Image, TouchableOpacity } from 'react-native';
+import * as Google from 'expo-google-app-auth';
+import firebase from 'firebase';
+import styles from './styles';
+
+
+export default class LoginScreen extends Component {
+   onSignIn = (googleUser) => {
+     // We need to register an Observer on Firebase Auth to make sure auth is initialized.
+     const unsubscribe = firebase.auth().onAuthStateChanged((firebaseUser) => {
+       unsubscribe();
+       // Check if we are already signed-in Firebase with the correct user.
+       if (!this.isUserEqual(googleUser, firebaseUser)) {
+       // Build Firebase credential with the Google ID token.
+         const credential = firebase.auth.GoogleAuthProvider.credential(
+           googleUser.idToken,
+           googleUser.accessToken
+         );
+         // Sign in with credential from the Google user.
+         firebase
+           .auth()
+           .signInWithCredential(credential).then(() => {
+           })
+           .catch((error) => {
+           // Handle Errors here.
+             const errorCode = error.code;
+             const errorMessage = error.message;
+             // The email of the user's account used.
+             const email = error.email;
+             // The firebase.auth.AuthCredential type that was used.
+             const credential = error.credential;
```

Comment:
you catch the errors here, however, they are not handled. Was there a plan to use these newly declared variables for something in particular? Otherwise, it appears that some variables are defined but never used

Reply:
Fixed!